From e868d4ecb9fb54ea602b090bdbd10d8fbb5ea53c Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 9 Feb 2018 16:09:13 +0800
Subject: [PATCH] ARM: rockchip_defconfig: remove CONFIG_PM_AUTOSLEEP

6e30a9a158bd ("remove CONFIG_PM_AUTOSLEEP from android-base.cfg")

Autosleep is no longer used by Android.

Change-Id: Ia024dbb6343abd8e1016febc08251dc5dc7badc1
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 78c5bf63feb3..93f4ad05101e 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -72,7 +72,6 @@ CONFIG_VFP=y
 CONFIG_NEON=y
 CONFIG_KERNEL_MODE_NEON=y
 # CONFIG_COREDUMP is not set
-CONFIG_PM_AUTOSLEEP=y
 CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
 # CONFIG_PM_WAKELOCKS_GC is not set
-- 
2.35.3

